src/components/pages/ParentFamily/ParentFamilyHome.js

```diff
@@ -2,15 +2,19 @@ import React, { useState, useEffect } from 'react';
 import axios from 'axios';
 import Banner from '../../common/Banner';
 import ParentSidebar from '../ParentHome/ParentSidebar';
-// import CreateNewStudent from './CreateNewStudent';
-// import { useParams } from 'react-router';
-// import CurrentCoursesDetails from './CurrentCoursesDetails';
+import CreateNewStudent from './CreateNewStudent';
 import '../../../styles/ParentStyles/index.less';
+import { Layout, Modal, Button } from 'antd';
+import 'antd/dist/antd.css';
 
 const ParentFamilyHome = () => {
+  const { Content, Sider } = Layout;
   const [studentInfo, setStudentInfo] = useState(null); //eslint-disable-line
-  // const [modal, setModal] = useState(false);
-  // const [modal2, setModal2] = useState(false);
+  const [addChildVisible, setAddChildVisible] = useState(false);
+  const [addChildConfirmLoading, setAddChildConfirmLoading] = useState(false);
+  const [addChildModalText, setAddChildModalText] = useState(
+    <CreateNewStudent />
+  );
 
   useEffect(() => {
     const token = JSON.parse(localStorage.getItem(`okta-token-storage`));
```

Comment:
Please don't refactor immediately but peeking into LocalStorage for an OktaToken is an anti-pattern as they've provided a `hook` called `useOktaAuth` that you can use to pull the token out of `Context`.